src/main/java/com/palechip/hudpixelmod/HudPixelConfigGui.java

```diff
@@ -22,10 +23,12 @@
         // these categories have to be there always
         list.addAll(new ConfigElement(configFile.getCategory(Configuration.CATEGORY_GENERAL)).getChildElements());
         list.addAll(new ConfigElement(configFile.getCategory(HudPixelConfig.DISPLAY_CATEGORY)).getChildElements());
-        if(HudPixelMod.instance().gameDetector.getCurrentGame() != null) {
+        if (HudPixelMod.instance().gameDetector.getCurrentGame() != null) {
             // Add the current game
             list.addAll(new ConfigElement(configFile.getCategory(HudPixelMod.instance().gameDetector.getCurrentGame().getConfigCategory())).getChildElements());
-            new ChatMessageComposer("Opened the config for currently played game. Open in lobby to access all options.", EnumChatFormatting.DARK_GREEN).send();
+            if (HypixelNetworkDetector.isHypixelNetwork) {
+                new ChatMessageComposer("Opened the config for currently played game. Open in lobby to access all options.", EnumChatFormatting.DARK_GREEN).send();
```

Comment:
This crashed the game, didn't it? I noticed it when developing but I forgot to fix it. Thanks.